core/components/organisms/grid/__stories__/index.story.jsx

```diff
@@ -43,11 +44,7 @@ export const all = () => {
   const separator = false;
 
   return (
-    <div
-      style={{
-        height: '350px',
-      }}
-    >
+    <div className="Grid-Wrapper">
```

Comment:
Please use a custom class name, `Grid-Wrapper` is a component style class name https://github.com/innovaccer/design-system/blob/master/css/src/components/grid.css#L3 .